fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/query/FDBNestedRepeatedQueryTest.java

```diff
@@ -1511,6 +1522,93 @@ void sumWholeRecordValueWithDuplicatesByKey() {
         }, this::querySumIntValueForRecordByKey);
     }
 
+    @DualPlannerTest(planner = DualPlannerTest.Planner.CASCADES)
+    void sumByKeyAndOther() {
```

Comment:
This test used to throw a protobuf validation error before this change. The index matchcandidate for the `sumValueByKeyAndOther` would create a result value that had two fields with protobuf index 1, hence the error.